### ISSUE TITLE
Am I using the right strain energy density function?

### ISSUE THREAD

**Issue by msnhao** [dilevin/CSC417-a3-finite-elements-3d#13]
Just to confirm, the function I'm supposed to use is:
W = C(I1 - 3) + D(J - 1)^2
Using the dW/dF given by Matlab, I'm getting ridiculously high forces while at rest. My F is close to identity at rest (with non-diagonal values of ~1e-16), but if I look at the equations spit out from Matlab, the forces can't equal 0 when F is identity so my bunny keeps exploding.

I've already taken into account that Matlab flattens according to columns and that my linearly implicit euler was working on the previous project.

**Comment by dilevin** [comment]
yes this is the correct strain energy density function and is the one implemented in the solution.

**Comment by msnhao** [comment]
Just to be doubly sure, I1 = tr(F^T*F), J = det(F), and it's not the same as what was done here: https://www.youtube.com/watch?v=fqGNoUV67lI

**Comment by abhimadan** [comment]
Please follow the formula in the video. It looks like this page has changed since the last time this course was offered, so now the I_1 we use is actually I_1 bar on the wikipedia page. Sorry about the confusion! We're just going to add it to the readme to avoid this issue in the future.
